### ISSUE TITLE
Don't lock keyboard input

### ISSUE THREAD

**Issue by MayorUshanka** [egaga/openvim#82]
The tutorial forces a pace on you by locking you into keystrokes you can type. While I was trying to get familiar with the commands and hjkl etc, the tutorial had already moved on and was no longer accepting my input.

**Comment by unelma** [comment]
Yes, there's a lot of low-hanging improvements to be made if I had the interest.
While were at it--and I know it's altogether a different thing--you can play around in https://openvim.com/sandbox.html

Thanks for the feedback.

**Comment by lefuturiste** [comment]
Yes, I had the exact same feeling, it's very frustrating because you can't really get familiar with the keys. We need time and practice to remember :smile: 
May be you can only allow the keys that are needed and let the player play with it and still wait for enter to complete the section ? (My suggestion)

**Comment by unelma** [comment]
It's a good idea, I think you're not the only one requesting that.
But but. I don't think I will put any time into OpenVim, but if I will, perhaps this kind of thing would be cool.
